tests/v1/test_password_reset_email.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from httpx import AsyncClient
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from ...main import app
+from api.db.database import Base, get_db
+from api.utils.settings import settings
+
+test_db_name = 'test_db'
+test_db_pw = 'root'
+
+# Create a test database engine
+SQLALCHEMY_DATABASE_URL = f"{settings.DB_TYPE}://postgres:{test_db_pw}@localhost:5432/{test_db_name}"
+engine = create_engine(SQLALCHEMY_DATABASE_URL)
+TestingSessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+# Override the get_db dependency to use the test database
+def override_get_db():
+    try:
+        db = TestingSessionLocal()
+        yield db
+    finally:
+        db.close()
+
+app.dependency_overrides[get_db] = override_get_db
+
```

Comment:
Please find a better way to do this. Look into mocking of database for testing. This test would fail except if we have all created `test_db`